stock_prices/stock_prices.py

```diff
@@ -3,7 +3,15 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  max_profit = 0
+  for i, buy_price in enumerate(prices):
```

Comment:
Oh I didn't see you were using a counter for each loop so that's why you were using enumerate.